plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -115,7 +115,7 @@ public BaseJdbcClient(
             Set<String> jdbcTypesMappedToVarchar,
             IdentifierMapping identifierMapping,
             RemoteQueryModifier remoteQueryModifier,
-            boolean supportsRetries)
+            @SupportsRetries boolean supportsRetries)
```

Comment:
`BaseJdbcClient` is never directly instantiated (it is abstract) so this annotation should not be here.
The constructor should also probably be `protected`

Reply:
removed the entire commit